exporters/dsexporter/dsexporter.go

```diff
@@ -1,39 +1,109 @@
 package main
 
 import (
+	"context"
+	"encoding/json"
 	"fmt"
 	"net/http"
+	"os"
+	"strings"
 
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"k8s.io/client-go/kubernetes"
+	// "k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
 )
 
+const service = "https://console-openshift-console.apps.stone-prd-rh01.pg1f.p1.openshiftapps.com/k8s/ns/appstudio-grafana/services"
+const check = "prometheus-appstudio-ds"
+var getInClusterConfig = config.GetConfigOrDie
+var getNewForConfig = kubernetes.NewForConfigOrDie
+
 type CustomCollector struct {
-	requestCounter prometheus.Counter
+	konfluxDSExporter *prometheus.GaugeVec
 }
 
 // Creating a new instance of CustomCollector.
 func NewCustomCollector() *CustomCollector {
 	return &CustomCollector{
-		requestCounter: prometheus.NewCounter(prometheus.CounterOpts{
-			Name: "request_count",
-			Help: "Number of requests handled by the handler",
-		}),
+		konfluxDSExporter: prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Name: "konflux_dsexporter",
```

Comment:
I don't think it makes sense using the `exporter` keyword in a metric name. @amisstea worked on the ticket for the metrics specs, and looking at [the outcome](https://docs.google.com/document/d/193FO8DYP8zCzzued53DiIT84xmyTm51v_OqiWzQi8Bg/edit?usp=sharing), I see that the example there was using the `probe` keyword.

Alex, do you think we should use the `probe` keyword? Any other suggestions?

Reply:
As discussed in our paired programming call, let's use `konflux_up` for the metric name (inspired by the prometheus `up` metric). I'll update that spec doc.